### ISSUE TITLE
Predict module not giving expected output

### ISSUE THREAD

**Issue by Soorya19Pradeep** [mehta-lab/VisCy#102]
The output images generated by the prediction module are not as expected. Here is the snapshot of the prediction from the 2D infection classifier trained for BJ5 cells generated using the test module:
![image](https://github.com/mehta-lab/VisCy/assets/101817974/20db2ec5-5c3e-406c-b4bb-125352d3a2b6)
This is the ground truth image of the infection score:
![image](https://github.com/mehta-lab/VisCy/assets/101817974/d8da1802-f06c-41fa-9d43-b26f5152f62d)
Here is the output from the prediction module:
![image](https://github.com/mehta-lab/VisCy/assets/101817974/d4282d5b-d3fc-46ab-baa8-03c782525694)
@ziw-liu, where do you think this can originate, and can you please help me debug this? Thanks!

**Comment by mattersoflight** [comment]
@Soorya19Pradeep please link the path to the module here. Thanks for flagging this!

**Comment by Soorya19Pradeep** [comment]
Sure @mattersoflight . The problem seems to come from the prediction module in my code [here](https://github.com/mehta-lab/VisCy/blob/418d6d9bd8d0a901068341f57126e9486c543c84/applications/infection_classification/classify_infection_2D.py#L243). The same line in the test module [here](https://github.com/mehta-lab/VisCy/blob/418d6d9bd8d0a901068341f57126e9486c543c84/applications/infection_classification/classify_infection_2D.py#L260) gives the right result (first image above). 
I will inspect if the source fed to the model in each of these modules and update the issue.

**Comment by Soorya19Pradeep** [comment]
I am closing the issue here. The issue has been resolved. 
There were two issues here. The issue with the prediction was already resolved when I recreated a fresh conda environment using a fresh installation of VisCy. 
Secondly, the issue seemed to persist as napari did not show me the new results. The wrong result kept showing even when I reopened a fresh napari window and created new zarr stores. On loading the same zarr today with a new OnDemand session it shows the expected result.
